lib/rubocop/config_loader.rb

```diff
@@ -113,6 +113,11 @@ def configuration_from_file(config_file)
         config = load_file(config_file)
         return config if config_file == DEFAULT_FILE
 
+        if ConfigLoader.ignore_parent_exclusion
```

Comment:
For consistency, call `ignore_parent_exclusion?` with no `ConfigLoader` receiver.